### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @danehansen

### CODE REVIEW

**Comment by danehansen** [comment]
:fish:

File: src/server.jsx [hunk 1]
```diff
@@ -56,7 +57,7 @@ class Server {
     var server = koa();
     server.keys = config.keys;
 
-    server.use(csrf.middleware);
+    //server.use(csrf.middleware);
```
Comment (JordanMilne) [inline src/server.jsx]: Were there CSRF token validation errors on the login form?

Comment (ajacksified) [inline src/server.jsx]: Whoops, forgot to fix this. Will add this back before merge.

**Comment by ajacksified** [comment]
:eyeglasses: @JordanMilne 

Re-added CSRF (check server.jsx).

**Comment by JordanMilne** [comment]
:fish:

**Pull request merged**